airflow/operators/bash_operator.py

```diff
@@ -83,18 +75,19 @@ def execute(self, context):
         """
         self.log.info("Tmp dir root location: \n %s", gettempdir())
 
-        airflow_home_value = conf.get('core', AIRFLOW_HOME_VAR)
-        pythonpath_value = os.environ.get(PYTHONPATH_VAR, '')
-
-        bash_command = ('export {}={}; '.format(AIRFLOW_HOME_VAR, airflow_home_value) +
-                        'export {}={}; '.format(PYTHONPATH_VAR, pythonpath_value) +
-                        self.bash_command)
-        self.lineage_data = bash_command
+        # Prepare env for child process.
+        if self.env is None:
+            self.env = os.environ.copy()
```

Comment:
This changes behaviour of the BashOperator. If the `AIRFLOW_HOME_VAR` or `PYTHONPATH_VAR` is set in the config instead of using an environment variable, this won't be picked up.

Reply:
The original reason we retrieve `AIRFLOW_HOME_VAR` from the config is that the executor doesn't pass environment variable down to the subprocess correctly and that is fixed by [adding `-E` option.](https://github.com/apache/incubator-airflow/commit/354492bc597130f43c76e7bec4bc894fb6deb7fe) `PYTHONPATH_VAR ` is always available in the env var mapping and we were retrieving it from env var mapping before this change.